### PR TITLE
Bug 1814692 - openshift-apiserver should create stable backend

### PR DESCRIPTION
**Pull request by sanchezl**
Excessive events like this:
```
Mar 14 04:38:32.591 I ns/openshift-apiserver-operator deployment/openshift-apiserver-operator Updated storage urls to https://10.0.0.3:2379,https://10.0.0.4:2379,https://10.0.0.5:2379 (3001 times)
```

### CODE REVIEW

File: pkg/operator/configobservation/etcdobserver/observe_etcd.go [hunk 1]
```diff
@@ -87,6 +87,7 @@ func ObserveStorageURLs(genericListers configobserver.Listers, recorder events.R
 		errs = append(errs, err)
 	}
 
+	sort.Strings(currentEtcdURLs)
```
Comment (sttts) [inline pkg/operator/configobservation/etcdobserver/observe_etcd.go]: this line is not necessary I think. But doesn't harm either.

**Comment by stlaz** [comment]
/close
fixed in https://github.com/openshift/cluster-openshift-apiserver-operator/pull/338

**Pull request closed without merging**